ra-optimisation/rao-commons/src/main/java/com/farao_community/farao/rao_commons/RaoUtil.java

```diff
@@ -87,20 +87,16 @@ public static void checkParameters(RaoParameters raoParameters, RaoInput raoInpu
         }
     }
 
-    public static SystematicSensitivityInterface createSystematicSensitivityInterface(RaoParameters raoParameters, RaoData raoData) {
+    public static SystematicSensitivityInterface createSystematicSensitivityInterface(RaoParameters raoParameters, RaoData raoData, boolean withPtdfSensitivities) {
```

Comment:
raoParameters is not sufficient here ? Not sure there is a need for added a new argument

Reply:
Unfortunately it does not only depend on the contents of RaoParameters. This method is called in 3 places : 
- in the LinearOptimizer: then we should compute PTDFs if approximation level is ..._AND_PST
- in the Leaf, for the initial evaluation: then we should compute PTDFs if approximation level is UPDATE_PTDF_WITH_TOPO or UPDATE_PTDF_WITH_TOPO_AND_PST 
- in the Leaf, when creating the IteratingLinearOptimizer: then we should compute PTDFs if approx level is ..._AND_PST like in the linear optimizer case

Reply:
ok I see :) 
There might be some duplicated code in the InitialSensitivityAnalysis that could be avoided now thanks to your new method signature. If you could have a look :angel: